### DISCUSSION TITLE
googlemap integration

### DISCUSSION THREAD

**Discussion by kttary** [ngageoint/geopackage-ios#44] [Ideas]
can we use googlemap instead of mapkit with geopackage-ios?

**Comment by bosborn** [comment]
We have not tried, but you should be able to use the library within Maps SDK for iOS.  You'd need to implement your own tile layers and shape conversions.  This would probably be a combination of the geopackage-ios implementations combined with the geopackage-android-map logic.

**Comment by bosborn** [comment]
_Converted from an Issue to a Discussion_

**Comment by kttary** [comment]
even more wonderful if geopackage available as Flutter plugin. We are move to flutter currently and missed this geopackage plugin
